descriptions/api.github.com/api.github.com.json

```diff
@@ -34831,6 +34825,79 @@
         }
       }
     },
+    "/repos/{owner}/{repo}/lfs": {
+      "put": {
+        "summary": "Enable Git LFS for a repository",
+        "description": "**Note:** The Git LFS API endpoints are currently in beta and are subject to change.",
```

Comment:
@imwiss We would usually add a required preview header for an endpoint like this. For SDK users there is currently no telling that `{DELETE,PUT} /repos/{owner}/{repo}/lfs` is subjected to change. Should we add a flag to `x-github`?